Documentation/ceph-upgrade.md

```diff
@@ -309,6 +309,15 @@ add below to the operator `env` variables.
       value: "/kubelet/path"
 ```
 
+If you want to enable Ceph Kernel clients on kernel < 4.17 which support quotas
```

Comment:
If this is set and you don't have at least kernel 4.17, you **won't** get quotas enforced. We should be clear on this.

Reply:
openSUSE's and SLE's kernel has backported quota support to a lower version (1.14 I think). So that's not exactly true. That's why this was implemented in the first place.